test/blackbox/annotations.bats

```diff
@@ -24,7 +24,8 @@ function setup_file() {
         "port": "8080"
     },
     "log": {
-        "level": "debug"
+        "level": "debug",
+        "output": "/tmp/zot.log"
```

Comment:
Use a file under `${BATS_FILE_TMPDIR}`, so that different tests are not writing the same file.